src/main/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/service/UserService.kt

```diff
@@ -95,7 +95,12 @@ class UserService(
   }
 
   @Transactional(readOnly = true)
-  fun findUsersAndEmails(): List<UserAndEmail> = userAndEmailRepository.findUsersAndEmails()
+  fun findUsersAndEmails(): List<UserAndEmail> {
+    log.info("Retrieving users and emails from database...")
+    val usersAndEmails = userAndEmailRepository.findUsersAndEmails()
+    log.info("Done retrieving users and emails from database...")
```

Comment:
Would it pay to include the number returned in the log message also?

Reply:
Thanks for the review Matt! Depending on the results of this quick test, I may end up drastically changing how this works (using streaming or pagination) so I'll leave as is for now